lib/client.js

```diff
@@ -1473,6 +1473,54 @@ MatrixClient.prototype.setPresence = function(opts, callback) {
 };
 
 /**
+* This is internal method
+* @param {MatrixClient} client
+* @param {string} opts
+* @param {module:client.callback} callback Optional.
```

Comment:
Docstrings are incorrect. For internal methods, I don't mind what level of documentation this has, but they should be correct if they are included at all. Please either remove these docs or make them correct.